src/awsexplorer/activation.ts

```diff
@@ -209,3 +219,8 @@ function updateAwsExplorerWhenAwsContextCredentialsChange(
         })
     )
 }
+
+async function copyArnToClipboard(arn: string) {
```

Comment:
Can you move this as `CopyArnCommand` to a separate file? It helps organize the commands for better reading.